modules/targetAcquisition/taxi/Taxi.py

```diff
@@ -95,16 +95,16 @@ def calculate_distance(pts):
         tempDistancesList = []
 
         # (Known) focal length of the camera in mm
-        focalLength = 1
+        focalLength = 24
 
         # (Known) real Height of the box in mm
         realHeight = 101.6
 
         # (Known) height of the image in pixels
-        imageHeight = 1
+        imageHeight = 1080
 
         # (Known) height of the sensor in mm
-        sensorHeight = 1
+        sensorHeight = 4.63
```

Comment:
Nitpick here, but would be nice if you could move these values into constant variables declared in CONSTANT_CASE (example of constant case there) at the top of the file.

Reply:
Changed to CONSTANT_CASE!